plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/internal/editor/tabbar/TabbarToolBarManager.java

```diff
@@ -214,7 +210,6 @@ private Color createColor(int color1, int color2, int ratio, Display display) {
 
     @Override
     public void dispose() {
```

Comment:
As explained orally, I prefer that you restore the order of methods calls (`super.dispose()` in first), as the change order has no link with the issue #81.

If you want, you can add another "cleanup" commit affter.

Reply:
Done